chainer/functions/__init__.py

```diff
@@ -99,6 +99,7 @@
 from chainer.functions.pooling import roi_pooling_2d
 from chainer.functions.pooling import spatial_pyramid_pooling_2d
 from chainer.functions.pooling import unpooling_2d
+from chainer.functions.subpixel import depth2space as depth2space_
```

Comment:
It is for the current ordering, but I was trying to protect against possible future reordering by others where

```
depth2space = depth2space.depth2space
Depth2Space = depth2space.Depth2Space
```
would throw an error.